### PR TITLE
Added cueMethod option to toggle or cycle through cuelist

### PR DESCRIPTION
**Pull request by JacobWasFramed**
Added cueMethod function. Default is a simple toggle. By defining the method attribute in your cue-button element in pgControls.vue and assigning value "list", triggering cue-button will send go command to MPC while cuelist is active, instead of release. default to method attribute is toggle and will behave that way if method is not defined.

### CODE REVIEW

File: src/cueButton.vue [hunk 1]
```diff
@@ -48,7 +51,7 @@ module.exports = {
 	},
 	methods: {
 		toggleCue: function() {
-			if (this.isActive()) {
+			if (this.isActive()&&this.cueMethod()!='list') {
```
Comment (jay-oswald) [inline src/cueButton.vue]: What about using a switch for the method insted? will make it easier to expand in the future and add more complicated methods if needed? Probably should break out the go and release to their own methods

Could do something like 
```
switch(this.method){
  case 'toggle':
    if(this.isActive())
      this.$root.sendCommand('cuelistRelease ' + this.$props.cue);
    else
      this.$root.sendCommand('cuelistGo ' + this.$props.cue);
    break;
  case 'list':
    this.$root.sendCommand('cuelistGo ' + this.$props.cue);
    break;
  default:
    console.log(`method ${this.method} is not valid`);
}
```

File: src/cueButton.vue [hunk 2]
```diff
@@ -68,6 +71,14 @@ module.exports = {
 				}
 			});
 			return active;
+		},
+		cueMethod: function() {
```
Comment (jay-oswald) [inline src/cueButton.vue]: Why is this method needed? should just be able to access the method via `this.method`

**Review by jay-oswald — CHANGES_REQUESTED** [review]

File: src/cueButton.vue (hunk 1 above)
Comment (JacobWasFramed) [reply src/cueButton.vue]: You're absolutely right. That's definitely cleaner and allows for expansion, as you said. I've added this to the code, tested, and committed the new version.

File: src/cueButton.vue (hunk 2 above)
Comment (JacobWasFramed) [reply src/cueButton.vue]: It's obsolete now with the switch function. It's been removed.

**Comment by JacobWasFramed** [comment]
Here is an updated commit with your changes. The updated code is tested and works without issue.

**Review by jay-oswald — APPROVED** [review]

**Pull request merged**